lib/pygments/popen.rb

```diff
@@ -215,7 +215,7 @@ def mentos(method, args=[], kwargs={}, original_code=nil)
 
       begin
         # Timeout requests that take too long.
-        timeout_time = ENV["MENTOS_TIMEOUT"] || 8
+	timeout_time = Integer(ENV["MENTOS_TIMEOUT"]) rescue 8
```

Comment:
The preferred invocation here would be `(ENV['MENTOS_TIMEOUT'] || 8).to_i`

Also the whitespace is broken as of the last commit (should be using spaces not tabs)